### DISCUSSION TITLE
Requirements.txt

### DISCUSSION THREAD

**Discussion by GuillermoBeaPepper** [j-convey/BankTextCategorizer#4] [General]
Hi, ive tried to use the proyect as it comes, without any modification and it was imposible to compile and execute it.
As the proyect doesnt have de requirements, i dont know what Version from each library should i install. 
Could you add the requirements.text to the proyect?
Im very interested in this example because it is something similar to a problem im trying to modeling.
Thank you in advance!

**Comment by tygertechforce** [comment]
I like the idea and functionality. I have Python3 installed and I followed your pip install instructions but have NO idea of how to run your program.  PLEASE HELP!
